src/get-started/flutter-for/ios-devs.md

```diff
@@ -5,467 +5,836 @@ description: Learn how to apply iOS developer knowledge when building Flutter ap
 
 <?code-excerpt path-base="get-started/flutter-for/ios_devs"?>
 
+{% assign sample_path = "get-started/flutter-for/ios_devs" %}
+
 This document is for iOS developers looking to apply
 their existing iOS knowledge to build mobile apps with Flutter.
 If you understand the fundamentals of the iOS framework
 then you can use this document as a
 way to get started learning Flutter development.
 
 {{site.alert.note}}
-  To integrate Flutter code into your iOS app, see
+  To integrate Flutter code into your **existing** iOS app, see
   [Add Flutter to existing app][].
 {{site.alert.end}}
 
-Before diving into this doc, you might want to watch a
-15-minute video from the [Flutter Youtube channel][] about
-the Cupertino package.
+<!-- Add this blurb once published -->
+<!-- Flutter is a framework for building cross-platform applications
+that uses the Dart programming language. 
+To understand some differences between programming with Dart
+and programming with Swift, see [Learning Dart as a Swift Developer][] 
+and [Comparing Dart Concurrency with Swift][]. -->
 
-<iframe width="560" height="315" src="{{site.youtube-site}}/embed/3PdUaidHc-E?rel=0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 Your iOS knowledge and skill set
-are highly valuable when building with Flutter,
+are highly valuable when building with Flutter, 
 because Flutter relies on the mobile operating system
 for numerous capabilities and configurations.
 Flutter is a new way to build UIs for mobile,
 but it has a plugin system to communicate
 with iOS (and Android) for non-UI tasks.
-If you're an expert in iOS development,
-you don't have to relearn everything to use Flutter.
+<!-- Add this sentence once published -->
+<!-- Learn more at [iOS and Apple hardware interactions with Flutter][] -->
```

Comment:
Is this another new page?

Reply:
Yes I will be adding another PR for this one